code/game/objects/items/reagent_containers/food/snacks.dm

```diff
@@ -3263,6 +3275,23 @@
 	reagents.add_reagent("nutriment", 7)
 	reagents.add_reagent("coco", 10)
 
+/obj/item/reagent_container/food/snacks/wrapped/chunk/hunk
+	name = "HUNK crate"
+	desc = "A 'crate', as the marketing called it, of \"The <b>HUNK</b>\" brand chocolate. An early version of the CHUNK box, the HUNK bar was hit by a class action lawsuit and forced to go into bankruptcy and get bought out by the Company when hundreds of customers had their teeth crack from simply attempting to eat the bar."
+	icon_state = "hunk"
+	w_class = SIZE_MEDIUM
+	hitsound = "swing_hit"
+	force = 35 //ILLEGAL LIMIT OF CHOCOLATE
```

Comment:
3 hit crit on unarmored
👏

Reply:
feature - there's a reason it was outlawed